.gitignore

```diff
@@ -14,3 +14,5 @@ pom.xml.asc
 .lein-repl-history
 /.cpcache
 /.clj-kondo
+*.iml
+/.idea/
```

Comment:
Can you remove this change, as it's not part of the reflection warnings fix.

Reply:
Done